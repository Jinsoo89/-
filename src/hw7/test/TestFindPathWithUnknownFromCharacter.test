# Find a path with Dijkstra's algorithm with unknown character 'from'

# Create graph players, which has a single node
CreateGraph players
AddNode players C.Ronaldo

# Find path between unknown node and known single node
FindPath players L.Messi C.Ronaldo