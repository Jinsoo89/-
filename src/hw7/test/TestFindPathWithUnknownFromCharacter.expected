# Find a path with Dijkstra's algorithm with unknown character 'from'

# Create graph players, which has a single node
created graph players
added node C.Ronaldo to players

# Find path between unknown node and known single node
unknown character L.Messi